app/Http/Controllers/AdminAuthController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Admin;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+
+class AdminAuthController extends Controller
+{
+    public function register(Request $request)
+    {
+        $fields = $request->validate([
+            'first_name' => array('required', 'string', 'max:40','regex:/(^[a-zA-Z])/u'),
+            'last_name' => array('required', 'string', 'max:40','regex:/(^[a-zA-Z])/u'),
+            'email' => array('required', 'string', 'unique:users,email', 'email', 'max:40'),
+            'password' => array('required', 'string', 'confirmed', 'max:40')
+        ]);
```

Comment:
There is inconsistency between your code and Previously merged code, Please unify how you are validating either using $this->validate or Validator Facade

Reply:
Ok, from now on, I will use $this->validate